src/nav/AppContainer.js

```diff
@@ -37,9 +37,9 @@ class AppContainer extends PureComponent {
   };
 
   handleConnectivityChange = isConnected => {
-    const { actions } = this.props;
+    const { actions, needsInitialFetch, isHydrated } = this.props;
     actions.appOnline(isConnected);
-    if (isConnected) {
+    if (isHydrated && !needsInitialFetch && isConnected) {
```

Comment:
needsInitialFetch  is just to make sure that `dispatch(realmInit(initData));` has been called as this make's sure that queue id saved to the state!